52.mender-gateway/docs.md

```diff
@@ -7,6 +7,54 @@ shortcode-core:
 github: false
 ---
 
+## mender-gateway 1.2.0
+
+_Released 12.28.2023_
+
+### Statistics
```

Comment:
For the private repos we don't publish the statistics, only the changelog.

Reply:
Follow-up here:
*  https://github.com/mendersoftware/mender-docs-changelog/pull/62